src/resource-metadata/common.js

```diff
@@ -0,0 +1,41 @@
+exports.schemaUrl = ""
+
+function extractArchitecture(architectures) {
+    const aws_arch = architectures?.[0]
+
+    return (aws_arch === 'x86_64')
+        ? 'amd64'
+        : (aws_arch === 'arm64')
+            ? 'arm64'
+            : ''
+}
```

Comment:
Two nested ternary operators are quite hard to understand.
I also don't see a point in mixing camelCase with snake_case
```suggestion
const extractArchitecture = architectures => {
    const awsArchitecture = architectures?.[0]
    switch (awsArchitecture) {
        case 'x86_64':
            return 'amd64'
        case 'arm64':
            return 'arm64'
        default:
            return ''
    }
};
```